src/trunk.c

```diff
@@ -4235,7 +4252,7 @@ trunk_flush_fullest(trunk_handle *spl, page_handle *node)
       }
       return trunk_flush(spl, node, fullest_pivot_data, FALSE);
    }
-   return FALSE;
+   return TRUE;
```

Comment:
Wow! Big BUG! I am worried that our test-/code-coverage is not up to snuff. 
We should really try to enhance the tests that are run in CI to trigger this kind of bug, to ensure that we get improved stability / coverage.

Reply:
This is a smaller bug than it appears for two reasons.

The first is that this function is only called when the node is full, so it always tries to flush. The only way to get to this return case is if the node flushes because pivots have too many branches and the node is no longer full. This doesn't really happen very much in practice.

The second is that returning false generally just causes the flush to be retried.